Tools/autotest/quadplane.py

```diff
@@ -78,6 +78,146 @@ def get_disarm_delay(self):
     def set_autodisarm_delay(self, delay):
         self.set_parameter("LAND_DISARMDELAY", delay)
 
+    def test_airmode(self):
+        """Check that plane.air_mode turns on and off as required"""
+        self.progress("########## Testing AirMode operation")
+        self.set_parameter("AHRS_EKF_TYPE", 10)
+        self.change_mode('QSTABILIZE')
+        self.wait_ready_to_arm()
+
+        """
+        SPIN_ARM and SPIN_MIN default to 0.10 and 0.15
+        when armed with zero throttle in AirMode, motor PWM should be at SPIN_MIN
+        If AirMode is off, motor PWM will drop to SPIN_ARM
+        """
+
+        self.progress("Verify that SERVO5 is Motor1 (default)")
+        motor1_servo_function_lp = 33
+        if (self.get_parameter('SERVO5_FUNCTION') != motor1_servo_function_lp):
+            raise PreconditionFailedException("SERVO5_FUNCTION not %d" % motor1_servo_function_lp)
+
+        self.progress("Verify that flightmode channel is 5 (default)")
+        default_fltmode_ch = 5
+        if (self.get_parameter("FLTMODE_CH") != default_fltmode_ch):
+            raise PreconditionFailedException("FLTMODE_CH not %d" % default_fltmode_ch)
+
+        """When disarmed, motor PWM will drop to min_pwm"""
+        min_pwm = self.get_parameter("Q_THR_MIN_PWM")
+
+        self.progress("Verify Motor1 is ast min_pwm when disarmed")
+        self.wait_servo_channel_value(5, min_pwm, comparator=operator.eq)
+
+        """set Q_OPTIONS bit AIRMODE"""
```

Comment:
Just a note that '# ' comments still work :-)  No action required.

Reply:
I just needed to change the color of # comments in my editor to be able to see them
Eyes are getting older and weaker :(